metrics/deploy-podman.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+# (Re)deploy Prometheus/Grafana to local podman, for development
+# Note: This keeps the prometheus-data volume
+set -eux
+MYDIR=$(realpath -m "$0"/..)
+
+# clean up old deployment
+podman kube down "${MYDIR}/metrics.yaml"
+for filter in grafana prometheus-config; do
+    podman volume ls -q --filter "name=$filter" | xargs --no-run-if-empty podman volume rm
+done
+
+# adjust the k8s deployment to work for podman play kube:
+#  - grafana-dashboards ConfigMap gets built dynamically in metrics/deploy-k8s.sh, but
+#    it is much more flexible and ergonomic to just mount the dashboards directory
+#  - replace grafana-admin secret with static "foobar" admin password
+patch -o- "$MYDIR/metrics.yaml" - <<EOF | podman kube play -
```

Comment:
I assume we cannot do an include?

Reply:
So seeing this works with secrets and `podman secret` being a thing, maybe we can use that instead?

That however does not solve the `hostPath` issue.

Reply:
https://www.redhat.com/sysadmin/podman-kubernetes-secrets

Reply:
Nice! I have an even higher opinion of `podman play kube` now, it has come a long a great way in the last few years.